models/commit_status.go

```diff
@@ -295,3 +252,8 @@ func ParseCommitsWithStatus(oldCommits *list.List, repo *Repository) *list.List
 	}
 	return newCommits
 }
+
+// HashCommitStatusContext hash context
+func HashCommitStatusContext(context string) string {
```

Comment:
This is only used in the models package. Please consider making it's visibility package-only.

Reply:
done.